Source/WTF/wtf/BitSet.h

```diff
@@ -397,11 +397,11 @@ inline constexpr bool BitSet<bitSetSize, WordType>::subsumes(const BitSet& other
     return true;
 }
 
-template<size_t bitSetSize, typename WordType>
-template<typename Func>
-ALWAYS_INLINE constexpr void BitSet<bitSetSize, WordType>::forEachSetBit(const Func& func) const
+template<typename WordType, typename Func>
+ALWAYS_INLINE constexpr void forEachSetBit(std::span<const WordType> bits, const Func& func)
```

Comment:
These should probably be moved to StdLibExtras.h

I think these could also be:
```
template<std::integral WordType>
constexpr void forEachSetBit(std::span<const WordType> bits, const Invocable<IterationStatus(size_t)> auto& func)
...


// Convenience wrapper for when you want to iterate the whole set.
template<std::integral WordType>
constexpr void forEachSetBit(std::span<const WordType> bits, const Invocable<void(size_t)> auto& func)
{
    forEachBitSet(bits, [&] (size_t i) {
        func(i);
        return IterationStatus::Continue;
    }
}
```
which would make the API a bit clearer. It might also be possible to implement the 0 start in terms of the `startIndex` version below? It seems like the compiler should optimize away the startIndex if it's a constant 0?

Reply:
Changed.
But for now, I don't want to change `startIndex` version separate as this patch's intent is not accidentally regressing performance.

Reply:
Ah, unfortunately, rather using Invocable here causes interface non-cleaner because compiler failed to identify which function it would like to use, and emitting ambiguous error. And it requires us to write all type signature explicitly in the caller side, and we would like to avoid this.

So, here, I'll just move this to StdLibExtras.h